scripts/config-dhcp.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+if dpkg-query -W -f='${Status}' isc-dhcp-server 1>/dev/null; then
+	echo "Already installed"
+	sudo apt-get upgrade isc-dhcp-server
```

Comment:
I believe that we should not upgrade, maybe add a max version allowed, usually the major number version says that the minor versions are binary and funcionalitty compatible.